extensions/gson/src/main/java/io/jsonwebtoken/io/GsonSerializer.java

```diff
@@ -0,0 +1,43 @@
+package io.jsonwebtoken.io;
```

Comment:
Per the same explanation of `GsonDeserializer`, please move this to `io.jsonwebtoken.gson.io`